editor/code/package.json

```diff
@@ -173,5 +191,11 @@
     "typecheck": "tsc -b",
     "compile": "npm run esbuild",
     "watch": "npm run esbuild -- --watch"
+  },
+  "__metadata": {
```

Comment:
Umm, still this keeps appearing here, but it should not, right?

Reply:
It's from `make ts-fmt`. It is quite annoying that compiling the project using `make extension` messes up all the formatting meaning I have to do `make ts-fmt` afterwards too, and that adds random crap here.

Reply:
I don't see any of that; but again you should never do `make extension` or `make ts-fmt` if you have vscode open.

I'd bet what you see here is a race between some watch target in vscode with some extensions you are using.

Reply:
BTW there is not way that make extension could modify any file in the source tree.